smac/facade/roar_facade.py

```diff
@@ -89,6 +91,20 @@ def __init__(self,
              success_states=[StatusType.SUCCESS, ],
              impute_censored_data=False, impute_state=None)
 
+        aggregate_func = average_cost
+        # initialize empty runhistory
+        if runhistory is None:
+            runhistory = RunHistory(aggregate_func=aggregate_func)
+        # inject aggr_func if necessary
+        if runhistory.aggregate_func is None:
+            runhistory.aggregate_func = aggregate_func
+
+        self.stats = Stats(scenario)
+        rs = RandomSearch(
+            acquisition_function=EI(model=model),
```

Comment:
Sorry, I don't understand why we have to pass EI into ROAR. ROAR should not have an model and hence, it should not use EI at all.

Reply:
The only real change here is that EI is now passed explicitly, before that it was passed implicitly in the super class (`SMACFacade`) and actually used during acquisition function optimization. Also, ROAR **currently does have a model**. However, I could simply change the PR to pass `None` as an acquisition function, then we could also remove the `RandomEPM` from ROAR and the whole SMAC3 code base. Sounds good?

Reply:
sounds very good.

Reply:
I changed the ROAR-part of the code. I left the `RandomEPM` in the code base as @KEggensperger reminded me that this is good for debugging.